bundle/edu.gemini.ags.servlet/src/main/java/edu/gemini/ags/servlet/ToContext.java

```diff
@@ -151,9 +151,10 @@ private TargetEnvironment targetEnv(HttpServletRequest req) throws RequestExcept
             // Here we just need an SPTarget with a contained NonSiderealTarget. In the end, this
             // is just used to determine whether it is a non-sidereal target or not in order to
             // pick the right guider.
+            // TODO: track down where this happens!
```

Comment:
Ok, I added a scheduling window and 1-element ephemeris to the non-sidereal target to be sure the coordinates are available. I think it's ok.